quickannotator/api/v1/image/routes.py

```diff
@@ -50,29 +55,187 @@ def delete(self, args):
         return 204
 
 #################################################################################
-@bp.route('/<int:project_id>/search', endpoint="image_search")
-class ImageSearch(MethodView):
-    @bp.arguments(server_models.SearchImageArgsSchema, location='query')
-    @bp.response(200, server_models.ImageRespSchema(many=True))
-    def get(self, args, project_id):
-        """     returns a list of Images
-        """
-        result = db_session.query(db_models.Image).filter(db_models.Image.project_id == project_id).all()
-        return result, 200
-
+    @bp.route('/<int:project_id>/search', endpoint="image_search")
+    class ImageSearch(MethodView):
+        @bp.arguments(server_models.SearchImageArgsSchema, location='query')
+        @bp.response(200, server_models.ImageRespSchema(many=True))
+        def get(self, args, project_id):
+            """     returns a list of Images
+            """
+            result = db_session.query(db_models.Image).filter(db_models.Image.project_id == project_id).all()
+            return result, 200
+    @bp.route('/project/<int:project_id>', endpoint="images")
```

Comment:
is this correcT? i think endpoint is only for internal usage and not externally visible, so would this be better as /project/{projid}/images?

Reply:
The **image** route has the endpoint set to ['image'](https://github.com/nanli-emory/QuickAnnotator/blob/b1bd445a27cc79d1812eb006410cd57be492bfce/quickannotator/api/v1/image/routes.py#L22), which means the URL for this API is {HOST}/image/project/{pro_id}.